traffic_ops/app/lib/API/Deliveryservice.pm

```diff
@@ -54,7 +54,10 @@ sub index {
 		$criteria{'me.logs_enabled'} = $logs_enabled ? 1 : 0;    # converts bool to 0|1
 	}
 
-	if ( !&is_privileged($self) ) {
+	my $tenant_utils = Utils::Tenant->new($self);
+	my $tenants_data = $tenant_utils->create_tenants_data_from_db();
+
+	if ( !&is_privileged($self) and !$tenant_utils->ignore_ds_users_table()) {
```

Comment:
Just saw your comment about adding these variables to "seed.sql".
It is possible and a matter of conventions.
For the "ignore-tenancy" I thought it should not be included as it is a "safety old mode knob" variable, that I'll might want to remove once the feature is deployed and we have confidence in